requirements.txt

```diff
@@ -1,8 +1,17 @@
 elasticsearch==7.16.2
 elasticsearch-dsl==7.4.1
-flask-cors
 iso3166
 json-merge-patch
 minio
 paho-mqtt
 pygeometa
+elasticsearch-dsl<8
```

Comment:
Is this required given line 2?

Reply:
nope it is not, I added another commit to remove it